js/interactive-guides/blueprint.js

```diff
@@ -44,10 +44,9 @@ var blueprint = (function(){
             stepContent.setCurrentStepName(stepContent.getStepNameFromHash(hash.substring(1)));
           } else {
             // no hash -> at the top of the guide
-            var hash = "#Intro";
-            accessContentsFromHash(hash);
+            accessContentsFromHash(location.hash);

```

Comment:
hash will be "". Does it still work using that?

Reply:
yes it does.  Fully tested.